webapi/Controllers/ChatHistoryController.cs

```diff
@@ -260,7 +260,7 @@ public async Task<IActionResult> GetAllChatSessionsAsync(string userId)
     {
         this._logger.LogInformation("Get imported sources of chat session {0}", chatId);
 
-        if (await this._sessionRepository.TryFindByIdAsync(chatId.ToString(), v => _ = v))
+        if (await this._sessionRepository.TryFindByIdAsync(chatId.ToString()))
```

Comment:
see also other comment: I believe we need to pass the PK, otherwise we're not addressing the perf problem. Either the client passes both values, or the "chat id" value should be a composite of PK+ID that's unpacked here

Reply:
TryFindByIdAsync is primarily called for the ChatSession entity.  This entity doesn't have a partition key distinct from the primary key.  As such nearly all calls to this method end up looking like: `TryFindByIdAsync(chatId, chatId)` .. which just reads a little redundant.  In the underlying code, the pk value is coallecesed as the partition key when not provided:

   ```
 public async Task<bool> TryFindByIdAsync(string id, string? partition = null, Action<T?>? callback = null)
    {
        try
        {
            callback?.Invoke(await this.FindByIdAsync(id, partition ?? id));
            return true;
        }
        catch (Exception ex) when (ex is ArgumentOutOfRangeException || ex is KeyNotFoundException)
        {
            return false;
        }
    }

```